.github/workflows/auto_build_operator.yml

```diff
@@ -0,0 +1,38 @@
+name: auto build operator
```

Comment:
change `operator` to `ctlmanager`will be better ? 
I think there will be more controllers in our mysql operator and here we make build the controllers.

Reply:
The name of our image is called mysql-operator, so I chose this name. whick one is  better? @andyli029